shared/getting-started/balenaCliAuth/_default.md

```diff
@@ -39,6 +40,7 @@ ID    APP NAME   DEVICE TYPE     ONLINE DEVICES DEVICE COUNT
 
 __Note__ See all the commands available with balenaCLI by running `balena help` 
 
-[cli]:https://github.com/balena-io/balena-cli
+[cli]:https://www.balena.io/docs/reference/cli/
```

Comment:
The balena.io CLI page (as opposed to the github.com CLI page) is composed of automatically extracted, selected sections from the CLI README.md file and also adds the CLI command reference, plus having links to the github repo (e.g. links INSTALL.md, FAQ, etc.) IMHO, it's a nicer page to use, both in usefulness and visual appearance, as a link for balenaCLI.

Reply:
Thanks. I need to pass through this guide again to update a number of links to use variables i.e. https://github.com/balena-io/docs/blob/master/config/links.coffee but I'll do this post-merge as this is a better update.